tests/baselines/reference/NonInitializedExportInInternalModule.js

```diff
@@ -41,7 +41,6 @@ var Inner;
 (function (Inner) {
     var ;
     let;
-    var ;

```

Comment:
All those inputs are incorrect in the first place. So I'm not quite bothered that they produce different kind of an output output. For an invalid input it's OK to emit an invalid output. It just happened that fixing a case for a valid input changed this here - but I think that's fine too. In a similar vein, I think it's OK to drop invalid statements from the invalid input.